http-gateway/swagger.yaml

```diff
@@ -19,6 +19,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/CloudConfiguration'
+  '/api/v1/oauth/callback':
+    get:
+      tags:
+        - 'Configuration'
+      summary: 'Converts OAuth redirect response to body'
+      description: |
+        Converts oauth redirect response to body as JSON. It used for getting device onboarding authorzation code as redirect.
```

Comment:
'It used' -> 'It is used'

'authorzation' -> 'authorization'
Weird that the spell-check didn't catch the second one, maybe I forgot to enable it for yaml files?

Reply:
Fixed